java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -694,7 +694,8 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_dayOfYear(JNIEnv *env,
 }
 
 JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_castTo(JNIEnv *env, jobject j_object,
-                                                                jlong handle, jint type) {
+                                                                jlong handle, jint type,
+                                                                jint scale) {
```

Comment:
We need to use the scale not just pass it in.

Reply:
done